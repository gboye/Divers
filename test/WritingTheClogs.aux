\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand*\HyPL@Entry[1]{}
\HyPL@Entry{0<</S/D>>}
\select@language{french}
\@writefile{toc}{\select@language{french}}
\@writefile{lof}{\select@language{french}}
\@writefile{lot}{\select@language{french}}
\HyPL@Entry{1<</S/D>>}
\@writefile{toc}{\contentsline {chapter}{\numberline {1}Débuts}{1}{chapter.1}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\@writefile{toc}{\contentsline {chapter}{\numberline {2}À vos marques, prêts, partez~!}{84}{chapter.2}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\@writefile{toc}{\contentsline {chapter}{\numberline {3}KasiMega}{219}{chapter.3}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\@writefile{toc}{\contentsline {chapter}{\numberline {4}La croix basque}{511}{chapter.4}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\@writefile{toc}{\contentsline {chapter}{\numberline {5}Premières berlinoises}{572}{chapter.5}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\@writefile{toc}{\contentsline {chapter}{\numberline {6}Bienvenus en Hongrie}{1382}{chapter.6}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\@writefile{toc}{\contentsline {chapter}{\numberline {7}Déluge de sigles}{1396}{chapter.7}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
