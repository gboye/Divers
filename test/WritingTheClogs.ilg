This is makeindex, version 2.15 [TeX Live 2017] (kpathsea + Thai support).
Scanning input file WritingTheClogs.idx.............done (9437 entries accepted, 0 rejected).
Sorting entries.................................................................................................done (139753 comparisons).
Generating output file WritingTheClogs.ind........done (830 lines written, 0 warnings).
Output written in WritingTheClogs.ind.
Transcript written in WritingTheClogs.ilg.
