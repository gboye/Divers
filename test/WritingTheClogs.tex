%!TEX TS-program = xelatex
% !TEX encoding = UTF-8 Unicode
% -*- coding: UTF-8; -*-
\documentclass{report}
\usepackage[breaklinks]{hyperref}
\usepackage[utf8]{inputenc}
\usepackage[french]{babel}
\usepackage{coloremoji-local}
\usepackage{tikzsymbols,hyperref,enumitem,ulem}
\usepackage{geometry,needspace,setspace}
\usepackage{makeidx}
\usepackage[font=footnotesize]{idxlayout}

\geometry{scale={0.8,0.88},marginparwidth=40pt} % scale={largeur,hauteur}
\onehalfspacing

\usepackage{fontspec}
\newfontfamily\sanskritfont [Script=Devanagari,Mapping=velthuis-sanskrit]{ITF Devanagari}
\newfontfamily\datafont {Helvetica}
\newfontfamily\greekfont {Times}

\definecolor{maroon}{HTML}{800000}
\definecolor{gold}{HTML}{FFD700}
\definecolor{navy}{HTML}{0000FF}
\definecolor{altwhite}{HTML}{F9F9F9}
\colorlet{white}{altwhite}

\setlistdepth{20}
\renewlist{itemize}{itemize}{20}
\setlist[itemize]{label=\textbullet}

\newcommand{\cacheNumber}[1]{}
\newcommand{\cacheName}[1]{\textbf{#1}\\}
\newcommand{\mileStone}[1]{\currentpdfbookmark{#1}{#1}\marginpar{\datafont\tiny #1}}
\newcommand{\cacheData}[1]{{\datafont\footnotesize #1}\\[-6pt]}
\newcommand{\Number}[1]{\##1}
\newcommand{\And}{\&}
\newcommand{\Underscore}{\_}
\newcommand{\Percent}{\%}
\newcommand{\Quoted}[1]{\lq\lq #1 \rq\rq}
\newenvironment{cacheText}{

}
{\bigskip

}
\renewenvironment{cacheText}{%
  \begin{list}{}{%
    \setlength{\topsep}{0pt}%
    \setlength{\leftmargin}{.5cm}%
    \setlength{\rightmargin}{.5cm}%
    \setlength{\listparindent}{\parindent}%
    \setlength{\itemindent}{\parindent}%
    \setlength{\parsep}{\parskip}%
  }%
  \item[]}{\end{list}\bigskip
  
  }

\setlength\parindent{0pt}
\title{{\Huge Writing the clogs}\\\#1--\#4000}
\author{\LARGE üêå Gamboy üåóüóù}
\date{Avril 2010 --- Juillet 2018\\[64pt]\includegraphics[scale=0.25]{WordCloud.png}}
\makeindex
\begin{document}

\maketitle

\input{TextesLog2}
\printindex
\end{document}

