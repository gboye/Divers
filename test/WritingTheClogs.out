\BOOKMARK [0][-]{chapter.1}{Débuts}{}% 1
\BOOKMARK [0][-]{64 Enfin~!!!.0}{64\040Enfin\040!!!}{}% 2
\BOOKMARK [0][-]{Premier contact.0}{Premier\040contact}{}% 3
\BOOKMARK [0][-]{Première WIG.0}{Première WIG}{}% 4
\BOOKMARK [0][-]{Cap sur les figuiers.0}{Cap\040sur\040les\040figuiers}{}% 5
\BOOKMARK [0][-]{Première cache en Asie.0}{Première cache en Asie}{}% 6
\BOOKMARK [0][-]{Visite à l'hôtel.0}{Visite à l'hôtel}{}% 7
\BOOKMARK [0][-]{Premier évent.0}{Premier évent}{}% 8
\BOOKMARK [0][-]{chapter.2}{À vos marques, prêts, partez !}{}% 9
\BOOKMARK [0][-]{Création du PeyoLand.0}{Création du PeyoLand}{}% 10
\BOOKMARK [0][-]{Visite à l'hôtel.0}{Visite à l'hôtel}{}% 11
\BOOKMARK [0][-]{Le jour du poisson.0}{Le\040jour\040du\040poisson}{}% 12
\BOOKMARK [0][-]{La fin du plat.0}{La\040fin\040du\040plat}{}% 13
\BOOKMARK [0][-]{Des coordonnées louches pour une belle vue.0}{Des coordonnées louches pour une belle vue}{}% 14
\BOOKMARK [0][-]{Premières fumées girondines.0}{Premières fumées girondines}{}% 15
\BOOKMARK [0][-]{La Chasse au Trésor.0}{La Chasse au Trésor}{}% 16
\BOOKMARK [0][-]{Attrapés par le landais.0}{Attrapés par le landais}{}% 17
\BOOKMARK [0][-]{Fin des Mystères niçois.0}{Fin des Mystères niçois}{}% 18
\BOOKMARK [0][-]{chapter.3}{KasiMega}{}% 19
\BOOKMARK [0][-]{Retour au pays des kiwis.0}{Retour\040au\040pays\040des\040kiwis}{}% 20
\BOOKMARK [0][-]{Première étape de montagne.0}{Première étape de montagne}{}% 21
\BOOKMARK [0][-]{chapter.4}{La\040croix\040basque}{}% 22
\BOOKMARK [0][-]{La croix basque est complète.0}{La croix basque est complète}{}% 23
\BOOKMARK [0][-]{chapter.5}{Premières berlinoises}{}% 24
\BOOKMARK [0][-]{Markdown débarque, une calamité !.0}{Markdown débarque, une calamité !}{}% 25
\BOOKMARK [0][-]{L'émergence des émojis.0}{L'émergence des émojis}{}% 26
\BOOKMARK [0][-]{Les bonus du Comté.0}{Les bonus du Comté}{}% 27
\BOOKMARK [0][-]{chapter.6}{Bienvenus\040en\040Hongrie}{}% 28
\BOOKMARK [0][-]{chapter.7}{Déluge de sigles}{}% 29
\BOOKMARK [0][-]{Le capharnaüm.0}{Le capharnaüm}{}% 30
\BOOKMARK [0][-]{Le 64.0}{Le\04064}{}% 31
